### ISSUE TITLE
implicitly supports the turn_on/turn_off methods without setting the proper ClimateEntityFeature

### ISSUE THREAD

**Issue by m4v3r1ckNl** [msvisser/remeha_home#62]
Logger: homeassistant.components.climate
Source: components/climate/__init__.py:365
integration: Climate ([documentation](https://www.home-assistant.io/integrations/climate), [issues](https://github.com/home-assistant/core/issues?q=is%3Aissue+is%3Aopen+label%3A%22integration%3A+climate%22))
First occurred: 21:20:37 (1 occurrences)
Last logged: 21:20:37

Entity None (<class 'custom_components.remeha_home.climate.RemehaHomeClimateEntity'>) implements HVACMode(s): off, heat, auto and therefore implicitly supports the turn_on/turn_off methods without setting the proper ClimateEntityFeature. Please create a bug report at https://github.com/msvisser/remeha_home/issues

**Comment by jellewie** [comment]
Can confirm

```
2024-09-16 17:00:01.747 WARNING (MainThread) [homeassistant.components.climate] Entity None (<class 'custom_components.remeha_home.climate.RemehaHomeClimateEntity'>) implements HVACMode(s): off, heat, auto and therefore implicitly supports the turn_on/turn_off methods without setting the proper ClimateEntityFeature. Please create a bug report at https://github.com/msvisser/remeha_home/issues
```

```
Logger: homeassistant.components.climate
Source: components/climate/__init__.py:370
integration: Climate (documentation, issues)
First occurred: 17:00:01 (1 occurrences)
Last logged: 17:00:01

Entity None (<class 'custom_components.remeha_home.climate.RemehaHomeClimateEntity'>) implements HVACMode(s): off, heat, auto and therefore implicitly supports the turn_on/turn_off methods without setting the proper ClimateEntityFeature. Please create a bug report at https://github.com/msvisser/remeha_home/issues
```

**Comment by msvisser** [comment]
Fixed with #59.
